exporter/exporterhelper/tracehelper_test.go

```diff
@@ -39,15 +40,13 @@ const (
 // TODO https://github.com/open-telemetry/opentelemetry-service/issues/266
 // Migrate tests to use testify/assert instead of t.Fatal pattern.
 func TestTraceExporter_InvalidName(t *testing.T) {
-	if _, err := NewTraceExporter("", newPushTraceData(0, nil)); err != errEmptyExporterName {
-		t.Fatalf("NewTraceExporter returns: Want %v Got %v", errEmptyExporterName, err)
-	}
+	_, err := NewTraceExporter("", newPushTraceData(0, nil))
+	require.Equalf(t, errEmptyExporterName, err, "NewTraceExporter returns: Want %v Got %v", errEmptyExporterName, err)
```

Comment:
you can achieve the same with `assert.Equal(t, errEmptyExporterName, err)` The print out of the error shows expected value and actual when this isn't true so no need to add the extra text.

Reply:
Just mentioned the same above...